<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="RowNotInTableException.xml" source-language="en-US" target-language="zh-TW">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5fe4b59957c717baaffe272111bd6a30b16f1d28d.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fe4b59957c717baaffe272111bd6a30b16f1d28d</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Data.RowNotInTableException">
          <source>Represents the exception that is thrown when you try to perform an operation on a <ph id="ph1">&lt;see cref="T:System.Data.DataRow" /&gt;</ph> that is not in a <ph id="ph2">&lt;see cref="T:System.Data.DataTable" /&gt;</ph>.</source>
          <target state="translated">表示嘗試在不是位於 <ph id="ph1">&lt;see cref="T:System.Data.DataRow" /&gt;</ph> 中的 <ph id="ph2">&lt;see cref="T:System.Data.DataTable" /&gt;</ph> 上執行作業時所擲回的例外狀況。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Data.RowNotInTableException">
          <source>The <ph id="ph1">&lt;xref:System.Data.RowNotInTableException&gt;</ph> is thrown when invoking the following methods on a row that has been deleted with either the <ph id="ph2">&lt;xref:System.Data.DataRow.Delete%2A&gt;</ph> or the <ph id="ph3">&lt;xref:System.Data.DataRowCollection.Remove%2A?displayProperty=nameWithType&gt;</ph> method.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Data.RowNotInTableException&gt;</ph>上叫用下列方法其中一種已刪除的資料列時，會擲回<ph id="ph2">&lt;xref:System.Data.DataRow.Delete%2A&gt;</ph>或<ph id="ph3">&lt;xref:System.Data.DataRowCollection.Remove%2A?displayProperty=nameWithType&gt;</ph>方法。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Data.RowNotInTableException">
          <source>The following example creates a <ph id="ph1">&lt;xref:System.Data.DataTable&gt;</ph> with one <ph id="ph2">&lt;xref:System.Data.DataColumn&gt;</ph> and 10 <ph id="ph3">&lt;xref:System.Data.DataRow&gt;</ph> objects.</source>
          <target state="translated">下列範例會建立<ph id="ph1">&lt;xref:System.Data.DataTable&gt;</ph>與一個<ph id="ph2">&lt;xref:System.Data.DataColumn&gt;</ph>和 10<ph id="ph3">&lt;xref:System.Data.DataRow&gt;</ph>物件。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Data.RowNotInTableException">
          <source>After deleting a row, the <ph id="ph1">&lt;xref:System.Data.DataRow.AcceptChanges%2A&gt;</ph> method is called causing the <ph id="ph2">&lt;xref:System.Data.RowNotInTableException&gt;</ph> to be thrown.</source>
          <target state="translated">刪除資料列之後,<ph id="ph1">&lt;xref:System.Data.DataRow.AcceptChanges%2A&gt;</ph>造成呼叫方法<ph id="ph2">&lt;xref:System.Data.RowNotInTableException&gt;</ph>擲回。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="T:System.Data.RowNotInTableException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Data.RowNotInTableException" /&gt;</ph> class without arguments.</source>
          <target state="translated">初始化沒有引數之 <ph id="ph1">&lt;see cref="T:System.Data.RowNotInTableException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:System.Data.RowNotInTableException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Data.RowNotInTableException" /&gt;</ph> class.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.Data.RowNotInTableException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.Data.RowNotInTableException.#ctor(System.String)">
          <source>The string to display when the exception is thrown.</source>
          <target state="translated">例外狀況擲回時要顯示的字串。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.Data.RowNotInTableException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Data.RowNotInTableException" /&gt;</ph> class with the specified string.</source>
          <target state="translated">使用指定的字串，初始化 <ph id="ph1">&lt;see cref="T:System.Data.RowNotInTableException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Data.RowNotInTableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The data that is required to serialize or deserialize an object.</source>
          <target state="translated">序列化或還原序列化物件所需要的資料。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Data.RowNotInTableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Description of the source and destination of the specified serialized stream.</source>
          <target state="translated">來源的描述和指定的序列化資料流的目的端。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Data.RowNotInTableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Data.RowNotInTableException" /&gt;</ph> class with serialization information.</source>
          <target state="translated">使用序列化資訊，初始化 <ph id="ph1">&lt;see cref="T:System.Data.RowNotInTableException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Data.RowNotInTableException.#ctor(System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">解釋例外狀況原因的錯誤訊息。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Data.RowNotInTableException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception, or a null reference (<ph id="ph1">&lt;see langword="Nothing" /&gt;</ph> in Visual Basic) if no inner exception is specified.</source>
          <target state="translated">造成目前例外狀況的例外狀況，若未指定內部例外狀況，則為 null 參考 (Visual Basic 中為 <ph id="ph1">&lt;see langword="Nothing" /&gt;</ph>)。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Data.RowNotInTableException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Data.RowNotInTableException" /&gt;</ph> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">使用指定的錯誤訊息以及造成此例外狀況的內部例外狀況的參考，初始化 <ph id="ph1">&lt;see cref="T:System.Data.RowNotInTableException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>